vms/components/multisig/camino_multisig.go

```diff
@@ -22,22 +25,78 @@ type Alias struct {
 	Owners verify.State        `serialize:"true" json:"owners"`
 }
 
-func (ma *Alias) InitCtx(ctx *snow.Context) {
-	ma.Owners.InitCtx(ctx)
+func ComputeAliasID(txID ids.ID) ids.ShortID {
```

Comment:
I'm very unsure about this func in this package
later, in tx execution, we'll define new alias id logic in tx execution
and in genesis alias id generation is completely different process